web_timeline/static/src/js/web_timeline.js

```diff
@@ -62,12 +62,17 @@ odoo.define('web_timeline.TimelineView', function (require) {
         //     });
         // },
 
-        parse_colors: function(){
-            if(this.fields_view.arch.attrs.colors) {
-                this.colors = _(this.fields_view.arch.attrs.colors.split(';')).chain().compact().map(function(color_pair) {
+        parse_colors: function () {
```

Comment:
@Creamaster PLZ you must avoid to change the formatting of the file even if this one is not conform to your criteria. It's hard to dissociate cosmetic changes from the changes required by your PR. These cosmetic changes should at least be done into a separate commit.

Reply:
I think this is for complying with JS linter, which is correct, but as said, better to keep it in a separate commit.

Reply:
As requested, done in separate commit